src/main/kotlin/app/revanced/patches/youtube/misc/openlinksdirectly/patch/OpenLinksDirectlyPatch.kt

```diff
@@ -25,7 +25,7 @@ import org.jf.dexlib2.iface.instruction.formats.Instruction35c
 @Patch
 @DependsOn([IntegrationsPatch::class, SettingsPatch::class])
 @Name("open-links-directly")
-@Description("Bypasses https://youtube.com/redirect URLs.")
+@Description("Bypass YouTube redirect URLs when you click a link in the video description.")
```

Comment:
Avoid saying "you" in descriptions:
```suggestion
@Description("Skips over redirection URLs to external links.")
```

Also mentioning YouTube shouldnt be necessary because the patch is meant for YouTube.